cmd/schema/main.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	profilesv1 "github.com/weaveworks/profiles/api/v1alpha1"
+	"github.com/weaveworks/schemer/definition"
+	schemapkg "github.com/weaveworks/schemer/schema"
+)
+
+func main() {
+	if len(os.Args) != 3 {
+		fmt.Println("Usage: schema <object-name> <outfile>")
+		os.Exit(1)
+	}
+	obj := os.Args[1]
+	outputFile := os.Args[2]
+
+	// generate the schema
+	schema, err := schemapkg.GenerateSchema("github.com/weaveworks/profiles/api", "v1alpha1", obj, false)
```

Comment:
I think the v1alpha1 is in the group thingy? But I might be wrong. :D

Reply:
ah yeh, meant to find a const or smth for that